pkg/handler/handler.go

```diff
@@ -537,7 +537,7 @@ func (h *Handler) PostApiV1OrganizationsOrganizationIDProjectsProjectIDIdentitie
 		return
 	}
 
-	if err := server.NewClient(h.client, h.namespace).SoftReboot(r.Context(), organizationID, projectID, identityID, serverID); err != nil {
+	if err := server.NewClient(h.client, h.namespace).Reboot(r.Context(), organizationID, projectID, identityID, serverID, true); err != nil {
```

Comment:
It seems to me this is less readable now, as you have a magic boolean, but it's not that big of a gripe.